middleware/csrf/memory.go

```diff
@@ -0,0 +1,78 @@
+package csrf
+
+import (
+	"errors"
+	"sync"
+	"time"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/utils"
+)
+
+type memoryStorage struct {
+	sync.RWMutex
+	tokens map[string]int64
+}
+
+// NewMemoryStorage - Creates new in-memory storage for CSRF tokens
+func NewMemoryStorage() fiber.Storage {
```

Comment:
I think it's better to make this factory method private

Reply:
what is the upside of making it private?

Reply:
I usually make stuff public only when it is necessary. We can change private to public but it's hard to do the opposite.

Reply:
i will make the change